packages/image_picker/image_picker_ios/example/lib/main.dart

```diff
@@ -216,63 +216,14 @@ class _MyHomePageState extends State<MyHomePage> {
     }
   }
 
-  Future<void> retrieveLostData() async {
```

Comment:
This was all Android-only code that I forgot to remove when copying the example during federation, so rather than fix the analysis issues with it I just removed it in this copy.